plugins/action_update_review_status/src/databaseMethods.ts

```diff
@@ -1,6 +1,23 @@
 import { ReviewStatus } from '../../../src/generated/graphql'
 
 const databaseMethods = (DBConnect: any) => ({
+  getReview: async (reviewId: number) => {
+    const text = `
+    SELECT
+      review_assignment_id AS "reviewAssignmentId"
+      FROM review
+      WHERE id = $1
+    `
+
+    try {
+      const result = await DBConnect.query({ text, values: [reviewId] })
+      return result.row[0]
+    } catch (err) {
+      console.log(err.message)
+      throw err
+    }
+  },
```

Comment:
Not sure why we need an extra database call. We already fetched all the associated reviews at the beginning, you should be able to pull it out of that result.

Reply:
Reverting this - as per convo in person

Also - I'll be adding a briefing issue on what we have agreed on